ui/app/appointments/views/manage/list/defaultListPrint.html

```diff
@@ -59,7 +59,9 @@ <h5 ng-if="!searchedPatient">{{::'APPOINTMENT_DATE' | translate}} {{::startDate
                     <th class="table-mid-width" ng-if="enableServiceTypes">{{::'APPOINTMENT_SERVICE_TYPE_FULL' |translate}}</th>
                     <th class="table-mid-width">{{::'APPOINTMENT_SERVICE_LOCATION_KEY' |translate}}</th>
                     <th>{{::'APPOINTMENT_STATUS' |translate}}</th>
-                    <!--<th class="notes">{{::'APPOINTMENT_COMMENTS' |translate}}</th>-->
+                    <!--<th>{{::'APPOINTMENT_ADDITIONAL_INFO' |translate}}</th>
+                    <th ng-repeat-start="columnValue in additionalInfoColumns" class="table-mid-width" ng-repeat-end>'APPOINTMENT_ADDITIONAL_INFO_COLUMN_'+columnValue.toUpperCase() \ translate</th>
```

Comment:
@buvaneswari-arun please remove the commented code

Reply:
As per Malini's comments during initial discussion, it is agreed that, we will retain the code as such as commented, so that the user can uncomment it and use if they want to add that particular column while they print the list.

Reply:
@buvaneswari-arun Indeed we can move this html to default-config so that whoever wants to override their changes, they can do it. May be we can have a card created for it.